app/models/organization.rb

```diff
@@ -357,7 +359,7 @@ def setup_available_statuses
   def get_available_statuses
     tmp_available_statuses = self.available_statuses.reject{|status| status.new_record?}
     statuses = []
-    if tmp_available_statuses.empty?
+    if tmp_available_statuses.empty? || tmp_available_statuses.collect{|status| status.status} == DEFAULT_STATUSES.collect{|k, v| k}
```

Comment:
#collect and #map are the same thing (https://stackoverflow.com/questions/5254732/difference-between-map-and-collect-in-ruby). I also think DEFAULT_STATUSES is an array.